src/components/Table.vue

```diff
@@ -1581,7 +1581,9 @@ export default {
     },
 
     initializeSort() {
-      const { enabled, initialSortBy, multipleColumns } = this.sortOptions;
+      const enabled = this.sortOptions.enabled;
+      const multipleColumns = this.sortOptions.multipleColumns;
+      const initialSortBy = this.sortOptions.initialSortBy || {};
```

Comment:
if this value is falsy, default to empty object to prevent `JSON.parse(undefined)` error